calcom/cal.com
# feat: Zapier For Teams

## What does this PR do?


Fixes https://github.com/calcom/cal.com/issues/7886



## Type of change



- New feature (non-breaking change which adds functionality)

## How should this be tested?

1) Set Up Zapier  (https://github.com/calcom/cal.com/blob/main/packages/app-store/zapier/README.md) and the booking created trigger as mentioned in readme.

2) Generate a zapier key for any team on /apps/zapier/setup and copy the key and use it for authentication on zapier

<img width="1439" alt="Screenshot 2023-07-01 at 1 13 57 AM" src="https://github.com/calcom/cal.com/assets/53316345/9a843c60-c87e-419f-bee0-78f3a0955e6b">

3) Now create a booking from your personal event type ( event type which is not under any team) and event type which belongs to the team you created the api key.

only event type belonging to the team should get triggered


<img width="819" alt="Screenshot 2023-07-01 at 1 47 51 AM" src="https://github.com/calcom/cal.com/assets/53316345/1c27da68-568c-4dd0-bed8-98a220e117e6">





## Mandatory Tasks
